tcms/utils/permissions.py

```diff
@@ -35,7 +10,7 @@ def initiate_user_with_default_setups(user):
     created user.
     """
     # create default permissions if not already set
-    assign_default_group_permissions()
+    Command.assign_default_group_permissions(assign_if_none=True)
```

Comment:
This needs to be a static method if you want to execute it without a class instance.

Reply:
It would be better to keep the verbosity - `self.stdout.write()`. Is it possible to instantinate the Command class without breaking tcms/utils/permissions.py?

Reply:
https://docs.python.org/3.6/library/functions.html?highlight=staticmethod#staticmethod